backend/data/src/main/kotlin/io/tolgee/repository/NotificationRepository.kt

```diff
@@ -0,0 +1,27 @@
+package io.tolgee.repository
+
+import io.tolgee.model.Notification
+import org.springframework.context.annotation.Lazy
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.stereotype.Repository
+
+@Repository
+@Lazy
+interface NotificationRepository : JpaRepository<Notification, Long> {
+  @Query(
+    """
+    SELECT n
+     FROM Notification n
+     LEFT JOIN FETCH n.project
```

Comment:
IMO, you don't need to fetch the project

Reply:
@JanCizmar Would you suggest to save only `projectId` instead of the project reference?

Or keep the project reference and just keep it lazy? I wouldn't do this because we need the project ID everytime so it would fetch the whole project. Or maybe I lack some hibernate skills.

Reply:
`SimpleProjectModel` fetching implemented.